floris/simulation/wake_velocity/gaussianModels/gauss_legacy.py

```diff
@@ -17,6 +17,95 @@
 from ..base_velocity_deficit import VelocityDeficit
 
 
+# from numba import njit
+
+
+# @njit
+def far_wake_start(D, Ct, yaw, alpha, beta, TI, x1):
+    """Determines where the far wake starts.
+
+    Args:
+        D (type) :
+        Ct (type) :
+        yaw (type) :
+        alpha (type) : `LegacyGauss.alpha`
+        beta (type) : `LegacyGauss.beta`
+        TI (type) :
+        x1 (type) : `turbine_coord.x1`
+
+    Returns:
+        x0 (type): starting point of the far wake.
+    """
+    ct = np.sqrt(1 - Ct)
+    x0 = D
+    x0 *= cosd(yaw)
+    x0 *= 1 + ct
+    x0 /= np.sqrt(2) * (4 * alpha * TI + 2 * beta * (1 - ct))
+    return x0 + x1
+
+
+# @njit
+def near_wake_velocity_deficit(x0, xR, x_locations, D, Ct, sigma_0):
+    """Determines where the far wake starts.
+
+    Args:
+        x0 (type) :
+        xr (type) :
+        x_locations (type) :
+        D (type) :
+        Ct (type) :
+        sigma_0 (type) : directional sigma
+
+    Returns:
+        sigma (type): velocity deficit of the near wake.
+    """
+    diff1 = x0 - xR
+    diff2 = x_locations - xR
+    sigma = (diff1 - diff2) / diff1
+    sigma *= 0.501 * D * np.sqrt(Ct / 2.0)
+    sigma += (diff2 / diff1) * sigma_0
+    return sigma
+
+
+# @njit
+def compute_r_C(
+    veer, sigma_y, sigma_z, y_locations, z_locations, delta, x2, yaw, D, HH, Ct
+):
+    """Computes r and C.
```

Comment:
What are `r` and `C`?

Reply:
I know I made this change, but I don't know, but those were the parameter calculations I was pulling out of the class. This piece, I believe, offered a decent amount of speedup because of how much it's called.